eta/modules/embed_vgg16.py

```diff
@@ -49,15 +57,28 @@ def __init__(self, d):
 
 
 class DataConfig(Config):
-    '''Data configuration settings.'''
+    '''Data configuration settings.
+
+    Inputs:
+        video_path (eta.core.types.Video): The input video
+
+    Outputs:
+        backing_path (eta.core.types.Directory): The directory in which to
+            write the embeddings
+    '''
 
     def __init__(self, d):
-        self.backing_path = self.parse_string(d, "backing_path")
         self.video_path = self.parse_string(d, "video_path")
+        self.backing_path = self.parse_string(d, "backing_path")
```

Comment:
Confused why these are swapped...

Reply:
`video_path` is the input and `backing_path` is the output, so my eyes want to see the input listed first

Reply:
Ah, I see.  They were originally alpha-sorted.  I suppose this is fine.